### PR TITLE
DTOS

### PR DESCRIPTION
**Pull request by mickamcia**
This is a single commit that divides DTOS into subfolders, also adds necessary DTOS for playlist api implementation

### CODE REVIEW

File: backend/YouTubeV2.Api.Tests/VideoControllerTests/AddVideoMetadataAsyncTests.cs [hunk 1]
```diff
@@ -42,8 +40,7 @@ public async Task Initialize()
         {
             _userServiceMock.Setup(x => x.GetByIdAsync(It.IsAny<string>())).ReturnsAsync(_user);
             _dateTimeProviderMock.Setup(x => x.UtcNow).Returns(_utcNow);
-            _blobImageService.Setup(x => x.UploadVideoThumbnailAsync(It.IsAny<string>(), It.IsAny<string>(), It.IsAny<CancellationToken>()));
```
Comment (KarolDenst) [inline backend/YouTubeV2.Api.Tests/VideoControllerTests/AddVideoMetadataAsyncTests.cs]: these changes are not connected to dtos and should be reverted

File: backend/YouTubeV2.Application/DTO/PlaylistDTOS/CreatePlaylistRequestDto.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using YouTubeV2.Api.Enums;
+
+namespace YouTubeV2.Application.DTO.PlaylistDTOS
+{
+    public record CreatePlaylistRequestDto(string name, Visibility visibility);
+
```
Comment (KarolDenst) [inline backend/YouTubeV2.Application/DTO/PlaylistDTOS/CreatePlaylistRequestDto.cs]: remove empty lines (in all new dtos)

File: backend/YouTubeV2.Application/Model/User.cs [hunk 3]
```diff
@@ -19,13 +19,5 @@ public User(RegisterDto registerDto)
             Name = registerDto.name;
             Surname = registerDto.surname;
         }
-
-        public User(string email, string nickname, string name, string surname)
```
Comment (KarolDenst) [inline backend/YouTubeV2.Application/Model/User.cs:23]: why remove this?

Comment (mickamcia) [reply backend/YouTubeV2.Application/Model/User.cs:23]: i don't know how it disappeared, added back

File: backend/YouTubeV2.Api.Tests/VideoControllerTests/AddVideoMetadataAsyncTests.cs (hunk 1 above)
Comment (mickamcia) [reply backend/YouTubeV2.Api.Tests/VideoControllerTests/AddVideoMetadataAsyncTests.cs]: i didn't touch this file, reverting

**Review by KarolDenst — APPROVED** [review]

**Pull request merged**